app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -735,6 +735,7 @@ class BrowserTabFragment : Fragment(), FindListener, CoroutineScope {
         omnibarTextInput.onFocusChangeListener =
             OnFocusChangeListener { _, hasFocus: Boolean ->
                 viewModel.onOmnibarInputStateChanged(omnibarTextInput.text.toString(), hasFocus, false)
+                if(!hasFocus)omnibarTextInput.hideKeyboard()
```

Comment:
NIT: better to use `hideKeyboardImmediately()` or `hideKeyboard()` that does the same as your code + request focus on a dummy view.

An additional note: please use curly brackets even if it's a single line of code, and ensure code follows our project style guide.

Reply:
@cmonfortep  just a question: why we request focus on a dummy view if this function is already requested in `hideKeyboard()`?